commit c814d82e5a435f93150cd28b7cf0f9d600ff400d
Author: Andrew John Hughes <gnu_andrew@member.fsf.org>
Date:   Fri Nov 2 18:08:52 2012 +0000

    PR55182: Fix build on RHEL6
    
    2012-11-02  Andrew John Hughes  <gnu_andrew@member.fsf.org>
    
    	PR classpath/55182
    	* configure.ac:
    	Lower required version to support autoconf
    	on RHEL6.  Make disabling Werror the default.
    	If a Qt4-specific moc is detected, use it
    	rather than the default.
    	* doc/Makefile.am:
    	Generate texi2dvi output in the build directory
    	and explicitly specify POSIX locale to work around
    	Debian bug 586134.
    	(clean-local): Remove output from texi2dvi.
    
    Signed-off-by: Andrew Hughes <gnu_andrew@member.fsf.org>

diff --git a/ChangeLog b/ChangeLog
index 03acce8..a88ea4f 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,17 @@
+2012-11-02  Andrew John Hughes  <gnu_andrew@member.fsf.org>
+
+	PR classpath/55182
+	* configure.ac:
+	Lower required version to support autoconf
+	on RHEL6.  Make disabling Werror the default.
+	If a Qt4-specific moc is detected, use it
+	rather than the default.
+	* doc/Makefile.am:
+	Generate texi2dvi output in the build directory
+	and explicitly specify POSIX locale to work around
+	Debian bug 586134.
+	(clean-local): Remove output from texi2dvi.
+
 2012-03-08  Andrew John Hughes  <ahughes@redhat.com>
 
 	* NEWS: Set correct release date.
diff --git a/configure.ac b/configure.ac
index bd48e84..b20a4bb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -37,7 +37,7 @@ AC_SUBST(CLASSPATH_MODULE)
 CLASSPATH_CONVENIENCE="-no-undefined"
 AC_SUBST(CLASSPATH_CONVENIENCE)
 
-AC_PREREQ(2.65)
+AC_PREREQ(2.63)
 AM_INIT_AUTOMAKE([1.9.0 gnu std-options tar-ustar -Wno-portability])
 AC_CONFIG_HEADERS([include/config.h])
 AC_PREFIX_DEFAULT(/usr/local/classpath)
@@ -143,9 +143,9 @@ AC_ARG_ENABLE([Werror],
               [case "${enableval}" in
                 yes) ENABLE_WERROR=yes ;;
                 no) ENABLE_WERROR=no ;;
-                *) ENABLE_WERROR=default ;;
+                *) ENABLE_WERROR=no ;;
               esac],
-              [ENABLE_WERROR=default])
+              [ENABLE_WERROR=no])
 
 dnl -----------------------------------------------------------
 dnl Default AWT toolkit
@@ -654,7 +654,11 @@ if test "x${COMPILE_JNI}" = xyes; then
 	QT_CFLAGS="$QT_CFLAGS -I$EXTRA_QT_INCLUDE_DIR",
 	AC_MSG_WARN([QWidget not found])))
 	AC_CHECK_PROG(MOC, [moc], [moc])
-	AC_CHECK_PROG(MOC, [moc-qt4], [moc-qt4])
+	AC_CHECK_PROG(MOC4, [moc-qt4], [moc-qt4])
+	if test x"$MOC4" != x ; then
+	  AC_MSG_NOTICE([Using $MOC4 as moc])
+	  MOC=$MOC4;
+	fi
     fi
     if test "x$HAVE_QT4" = "xno"; then
       AC_MSG_NOTICE([Looking for QT_CFLAGS and QT_LIBS without pkg-config])
diff --git a/doc/Makefile.am b/doc/Makefile.am
index 5ff03f0..eefbd3a 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -4,8 +4,9 @@ EXTRA_DIST = README.jaxp texi2pod.pl
 
 info_TEXINFOS = cp-hacking.texinfo cp-vmintegration.texinfo cp-tools.texinfo
 
+# POSIX locale necessary to make grep work; see http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=586134
 .texinfo.dvi:
-	texi2dvi $<
+	LC_ALL=POSIX texi2dvi --build-dir=$(builddir) -o $@ $<
 
 .dvi.ps:
 	dvips -o $@ $<
@@ -87,3 +88,5 @@ gjdoc.pod: $(srcdir)/cp-tools.texinfo
 
 CLEANFILES = $(TOOLS_MANFILES)
 
+clean-local:
+	-rm -rf *.t2d
